functions.php

```diff
@@ -27,4 +27,21 @@ function ppm_log($expression, $_ = null)
     ob_start();
     call_user_func_array('var_dump', func_get_args());
     file_put_contents('php://stderr', ob_get_clean() . PHP_EOL, FILE_APPEND);
+}
+
+function pcntl_enabled()
+{
+    if (!function_exists('pcntl_signal')) {
+        return false;
+    }
+
+    $requiredFunctions = array('pcntl_signal', 'pcntl_signal_dispatch', 'pcntl_fork');
```

Comment:
We can check on any pcntl_ prefixed function, but I'm not sure, that it is possible to enable them in some environments, so I propose to keep the list up to date.

Reply:
Does it really need `pcntl_fork`?